packages/mui-material/src/Popover/Popover.js

```diff
@@ -245,11 +245,19 @@ const Popover = React.forwardRef(function Popover(inProps, ref) {
       // Check if the vertical axis needs shifting
       if (top < marginThreshold) {
         const diff = top - marginThreshold;
-        top -= diff;
+
+        if (top >= -1) {
+          top -= diff;
+        }
+
```

Comment:
I understand that this is to "disable" the `marginThreshold` so the popover can move outside the window view, but I think it might be better to actually add an opt-in prop `"disableMarginThreshold"`. This way our change would be backward compatible and developers that are using the component won't see behavior changes unless opting in for the prop.